infra/modules/providers/azure/ml-workspace/tests/unit/ml_workspace_unit_test.go

```diff
@@ -0,0 +1,44 @@
+package unit
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	tests "github.com/microsoft/cobalt/infra/modules/providers/azure/ml-workspace/tests"
+	"github.com/microsoft/terratest-abstraction/unit"
+)
+
+var resourceCount = 1
+var workspace = "pwc-" + strings.ToLower(random.UniqueId())
```

Comment:
What is the meaning of the `pwc` prefix? If we omit the workspace all-together, the test harness should generate a random one for you. 

This would involve deleting this line + line 35

Reply:
Sorry, will fix it, it should be something like : `sample`